tests/processes/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .wps_test_process import TestProcess
+
+processes = [
+    TestProcess(),
+]
```

Comment:
I don't believe it is necessary to list the process(es) in here. In the structure of a bird project this file is used to collect all the available processes. Here though, we can just directly import `TestProcess` the way you do in `test_logging.py` and other files.